src/typing/type_inference_js.ml

```diff
@@ -5084,6 +5113,8 @@ and mk_class_elements cx instance_info static_info body = Ast.Class.(
       let this, super, method_sigs, getter_sigs, setter_sigs =
         if static then static_info else instance_info
       in
+      let yield = MixedT (mk_reason "no yield" loc) in
```

Comment:
Is this considering generator methods? It looks like you're not pivoting on the `generator` param of the Function expression above...

(Also mind adding a test or two for class methods in any case?)